src/calendar.js

```diff
@@ -1,310 +1,360 @@
 /*
-*  AngularJs Fullcalendar Wrapper for the JQuery FullCalendar
-*  API @ http://arshaw.com/fullcalendar/
-*
-*  Angular Calendar Directive that takes in the [eventSources] nested array object as the ng-model and watches it deeply changes.
-*       Can also take in multiple event urls as a source object(s) and feed the events per view.
-*       The calendar will watch any eventSource array and update itself when a change is made.
-*
-*/
+ *  AngularJs Fullcalendar Wrapper for the JQuery FullCalendar
+ *  API @ http://arshaw.com/fullcalendar/
+ *
+ *  Angular Calendar Directive that takes in the [eventSources] nested array object as the ng-model and watches it deeply changes.
+ *       Can also take in multiple event urls as a source object(s) and feed the events per view.
+ *       The calendar will watch any eventSource array and update itself when a change is made.
+ *
+ */
 
 angular.module('ui.calendar', [])
-  .constant('uiCalendarConfig', {calendars: {}})
-  .controller('uiCalendarCtrl', ['$scope', 
-                                 '$locale', function(
-                                  $scope, 
-                                  $locale){
-
-      var sources = $scope.eventSources,
-          extraEventSignature = $scope.calendarWatchEvent ? $scope.calendarWatchEvent : angular.noop,
-
-          wrapFunctionWithScopeApply = function(functionToWrap){
-              return function(){
-                  // This may happen outside of angular context, so create one if outside.
-
-                  if ($scope.$root.$$phase) {
-                      return functionToWrap.apply(this, arguments);
-                  } else {
-                      var args = arguments;
-                      var self = this;
-                      return $scope.$root.$apply(function(){
-                          return functionToWrap.apply(self, args);
-                      });
-                  }
-              };
-          };
-
-      var eventSerialId = 1;
-      // @return {String} fingerprint of the event object and its properties
-      this.eventFingerprint = function(e) {
-        if (!e._id) {
-          e._id = eventSerialId++;
+    .constant('uiCalendarConfig', {calendars: {}})

```

Comment:
Can you remove that or this is on purpose? @kamthamc this make unreadable what are real changes in the code and what is only change of formatting... I propose changing formatting in seperate PR if it's necessary.